content/docs/user-guide/troubleshooting.md

```diff
@@ -75,3 +75,19 @@ DVC only supports [symlinked files](/doc/command-reference/add#add-symlink) as
 valid targets for `dvc add`. If the target path is a directory symlink, or if
 the target path contains any intermediate directory symlinks, `dvc add` will
 fail.
+
+## No possible cache types {#cache-types}
```

Comment:
This title sounds like it has a typo but I'm not sure what we're trying to explain so can't suggest something yet (see https://github.com/iterative/dvc/pull/4801#pullrequestreview-521907242)